.cookiecutter/includes/tox/deps

```diff
@@ -0,0 +1,2 @@
+pluggy
+lint,tests: pytest-mock
```

Comment:
Our apps and libraries use a `patch` fixture of our own invention. Here's h-cookiecutter-pypackage's copy of it: <https://github.com/hypothesis/h-cookiecutter-pypackage/blob/b591cc4c6d6691cae4b6df8e4de8b685b333d0b6/tests/conftest.py#L7-L19>. But [pytest-mock](https://github.com/pytest-dev/pytest-mock/) is a pytest plugin that provides a mock fixture and is maintained by the pytest team. Maybe we should switch to that? The `pytest-mock` dependency could also be added to the cookiecutter so that individual projects don't need to add it like this.

Reply:
If does what we need, I'd say so. I'm been a big fan of using the standard stuff directly instead of our abstraction, but it's definitely handy some times. If this is a straight replacement for it, why not?

Reply:
There's a comment below, but I'm not sure why we've created this option to insert content into the `tox.ini` file when standardisation is possible without it.

It would also be very useful if the template highlighted where this came from as it's completely opaque when reading `tox.ini` that some of it is templated in, and most is standard.

I can see a general solution being a bit tricky as we can't always rely on injecting comments with '#' in them. The default includes could have the comments inside them and achieve much the same thing.

Should the path here be `tox.ini/deps`?

Reply:
Yeah I think `tox.ini/deps` would be a good change. Since that's a change to the cookiecutter I'm gonna postpone that until review of the cookiecutter itself